ui/src/criteria/attribute.tsx

```diff
@@ -45,6 +46,11 @@ class _ implements CriteriaPlugin<Data> {
   }
 
   generateFilter(entityVar: string, fromOccurrence: boolean) {
+    if (this.data.selected.length === 0) {
+      return null;
+    }
+    console.log(entityVar);
```

Comment:
before I finish the generateFilter. It will fail the test due to warning (do not use the parameters, entityVar, formOccurence).So just put 2 console there.

Reply:
For cases like this you should add a comment of the form:
```
// TODO(ginay): Implement generateFilter.
```

The important part is the TODO since that's a standard marker so we can find similar instances across the code.

You can also combine these into a single statement:
```
console.log(entityVar, fromOccurrence);
```